src/main/java/com/stuypulse/robot/MotorStall.java

```diff
@@ -0,0 +1,52 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package com.stuypulse.robot;
+
+import com.stuypulse.robot.subsystems.Funnel;
+
+import edu.wpi.first.wpilibj.Timer;
+
+public class MotorStall implements Runnable {
```

Comment:
I don't think the approach to this class is correct, you're making it so that this class would be using the Funnel and its' properties to get data, but I don't feel like this is good design. Instead you should make an interface which the Funnel subsystem will implement that forces that subsystem to have methods that we can use to get the encoder data, etc.